components/collector/tests/source_collectors/dependency_track/test_source_up_to_dateness.py

```diff
@@ -181,3 +188,12 @@ async def test_source_up_to_dateness_with_pagination(self):
         response = await self.collect(get_request_json_return_value=self.projects())
         self.assert_measurement(response, value="7", landing_url=self.LANDING_URL, entities=self.entities())
         DependencyTrackBase.PAGE_SIZE = default_size
+
+    async def test_source_up_to_dateness_with_reversed_direction(self):
+        """Test that the source up-to-dateness can be measured if the metric direction has been reversed."""
+        self.metric["direction"] = ">"
```

Comment:
Huh but what does `up-to-dateness` in a descending manner mean? That it's **not** supposed to be more up to date than a certain timeframe 🤔

Reply:
As mentioned above, there is currently no way to prevent users from changing the direction.